### PR TITLE
blog: security groups on ovn networks

### PR DESCRIPTION
**Pull request by maiqueb**
I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @maiqueb 

This pull request needs review by: 
@dominikholler 
@mmirecki

### CODE REVIEW

File: source/blog/2019-04-15-ovn-security-groups.html.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+In this post I will introduce and showcase how security groups can be used to
+enable certain scenarios.
+
+[Security groups](https://developer.openstack.org/api-ref/network/v2/#security-groups-security-groups) allow fine-grained access control to - and from - the oVirt VMs attached to external OVN networks.
+
+The Networking API v2 defines security groups as a white list of rules - the
+user specifies in it which traffic is allowed. That means, that when the rule
+list is empty, neither incoming nor outgoing traffic is allowed (from the VMs
+perspective).
+
```
Comment (dominikholler) [inline source/blog/2019-04-15-ovn-security-groups.html.md]: Embedding the first youtube video here would be nice.

File: source/blog/2019-04-15-ovn-security-groups.html.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+READMORE
+
+## Provided tools
```
Comment (dominikholler) [inline source/blog/2019-04-15-ovn-security-groups.html.md]: I am not sure if all the details are useful here, or if a link to your text on github would be sufficient, but the text does not hurt, as long as both videos are embedded above.

File: source/blog/2019-04-15-ovn-security-groups.html.md [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+
+# attach the web_clients group to the web_client VM
+ansible-playbook -i localhost update_ports.yml \
+  --extra-vars="port_uuid=<web_client_vm_id> sec_groups=<web_clients_group_id>"
+
+```
+
+**NOTE:** to find the security group IDs and the port IDs, use the [list_openstack_entities.py](list_openstack_entities.py) script - or the OpenStack CLI.
+
+## This on video
```
Comment (dominikholler) [inline source/blog/2019-04-15-ovn-security-groups.html.md]: I think this should be on top, and the first video should be shown in the summary.

File: source/blog/2019-04-15-ovn-security-groups.html.md [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+enable certain scenarios.
+
+[Security groups](https://developer.openstack.org/api-ref/network/v2/#security-groups-security-groups) allow fine-grained access control to - and from - the oVirt VMs attached to external OVN networks.
+
+The Networking API v2 defines security groups as a white list of rules - the
+user specifies in it which traffic is allowed. That means, that when the rule
+list is empty, neither incoming nor outgoing traffic is allowed (from the VMs
+perspective).
+
+READMORE
```
Comment (dominikholler) [inline source/blog/2019-04-15-ovn-security-groups.html.md]: Did you check if the READMORE is working? My local execution is not in the mood to work.

File: source/blog/2019-04-15-ovn-security-groups.html.md [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+
+To properly install openstacksdk on CentOS / RHEL, execute:
+```bash
+yum install -y epel-release
+yum install -y python-pip
+pip install --user -r requirements.txt
+```
+
+That will install openstacksdk from pip with the correct dependencies, which
+are listed in the [requirements.txt](requirements.txt) file.
```
Comment (dominikholler) [inline source/blog/2019-04-15-ovn-security-groups.html.md]: The repo internal links have to be updated or the whole details can be replaced by a link to https://github.com/maiqueb/ovirt-security-groups-demo/blob/master/README.md.

Comment (maiqueb) [reply source/blog/2019-04-15-ovn-security-groups.html.md]: Linked against the original blog post, hence, no longer needed.

File: source/blog/2019-04-15-ovn-security-groups.html.md (hunk 1 above)
Comment (maiqueb) [reply source/blog/2019-04-15-ovn-security-groups.html.md]: Done.

File: source/blog/2019-04-15-ovn-security-groups.html.md (hunk 2 above)
Comment (maiqueb) [reply source/blog/2019-04-15-ovn-security-groups.html.md]: Linked against hte original README.

**Comment by dominikholler** [comment]
@maiqueb  Thanks for the update.
I guess the blogpost would be more even more eye-catching, if the youtube video would be ebedded or shown as a thumbnail like `[IMAGE ALT TEXT HERE](http://www.youtube.com/watch?v=YOUTUBE_VIDEO_ID_HERE)` ? 
If you would not like to add the picture, I would ack this PR, too.

File: source/blog/2019-04-15-ovn-security-groups.html.md (hunk 3 above)
Comment (maiqueb) [reply source/blog/2019-04-15-ovn-security-groups.html.md]: Done

File: source/blog/2019-04-15-ovn-security-groups.html.md (hunk 4 above)
Comment (maiqueb) [reply source/blog/2019-04-15-ovn-security-groups.html.md]: Done

**Comment by maiqueb** [comment]
> Please see @dominikholler comments

@sandrobonazzola done.

**Pull request merged**